nautobot/core/tests/test_migration.py

```diff
@@ -0,0 +1,40 @@
+from django.apps import apps
```

Comment:
Did you add this? I recall there being something @HanlinMiao did with this class. IMO this should be in a library called `migrations.py` since it's publishing a base class.

Reply:
This is pulled over from Hanlin's branch. I think this should go in `nautobot.core.testing.migrations`

Reply:
I just changed the test to be in `nautobot.core.testing.migrations`